cli/cmd/brokers.go

```diff
@@ -19,16 +41,39 @@ var brokersCmd = &cobra.Command{
 	Long:  "Manage brokers associated with a service controller",
 }
 
-var createCmd = &cobra.Command{
+var createBrokersCmd = &cobra.Command{
 	Use:   "create",
 	Short: "Add a service broker to service controller",
 	Long:  "Add a service broker to service controller",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		return fmt.Errorf("Not implemented yet")
+		if len(name) == 0 {
```

Comment:
its a pet-peeve of mine, but, to me required args shouldn't require a flag. Can we make it so that its:
`sc brokers create NAME BROKER-URL`  ?